standalone-metastore/metastore-server/src/test/java/org/apache/hadoop/hive/metastore/client/TestTablesGetExists.java

```diff
@@ -286,13 +286,6 @@ public void testGetTables() throws Exception {
     Assert.assertTrue("Comparing tablenames", tables.contains(testTables[5].getTableName()));
     Assert.assertTrue("Comparing tablenames", tables.contains(testTables[6].getTableName()));
 
-    tables = client.getTables("*", "*");
-    Assert.assertEquals("All tables size", 7, tables.size());
-    tables = client.getTables("d*", "*");
-    Assert.assertEquals("All tables size", 7, tables.size());
-    tables = client.getTables("def*", "*");
-    Assert.assertEquals("All tables size", 5, tables.size());
```

Comment:
Remove these tests, discussed in https://github.com/apache/hive/pull/4326#issuecomment-2143551034.